src/ws/channel.rs

```diff
@@ -16,6 +18,12 @@ pub enum Channel {
     StopOrder,
 }
 
+impl std::fmt::Display for Channel {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{:?}", self)
```

Comment:
Since this implementation of `std::fmt::Display` is basically just a wrapper around `std::fmt::Debug`, I feel like at the moment we should just remove this implementation altogether, and use `std::fmt::Debug` directly instead.

I don't think this implementation is currently being used anyway?

Reply:
fixed